lib/src/mustachio/parser.dart

```diff
@@ -177,7 +177,7 @@ class MustachioParser {
   ///
   /// [_index] should be at the character immediately following the `#`
   /// character which opens a possible section tag.
-  _TagParseResult _parseSection({@required invert}) {
+  _TagParseResult _parseSection({@required bool invert}) {
```

Comment:
*cringe* yikes, thanks!

Reply:
@pq Is this something that `type_annotate_public_apis` should have caught?

Reply:
you'd think there should be a "no implicit dynamic" lint of some sort.   At least, whenever I write an implicit dynamic it's almost always an accident.